src/main/webapp/consentGroup/NewConsentGroup.js

```diff
@@ -225,7 +217,7 @@ class NewConsentGroup extends Component {
 
   showSubmit = (currentStep) => {
     let renderSubmit = false;
-    if (currentStep === 4) {
+    if (currentStep === 3) {
```

Comment:
Please change this number with a constant indicating last step, something like LAST_STEP_IDX or infer the value from array size ...

Reply:
Changed as suggested in New Project and New Consent Group.